modules/sandboxed-containers-collecting-data.adoc

```diff
@@ -6,26 +6,17 @@
 
 :_content-type: CONCEPT
 [id="sandboxed-containers-collecting-data_{context}"]
-= About collecting {sandboxed-containers-first} data
+= About {sandboxed-containers-first} log data
 
-You can use the `oc adm must-gather` CLI command to collect information about your cluster. The following features and objects are associated with {sandboxed-containers-first}:
+When you collect log data about your cluster, the following features and objects are associated with {sandboxed-containers-first}:
 
-* All namespaces and their child objects that belong to any {sandboxed-containers-first}
-resources
+* All namespaces and their child objects that belong to any {sandboxed-containers-first} resources
 * All {sandboxed-containers-first} custom resource definitions (CRDs)
 
-The `oc adm must-gather` CLI command collects the following component logs:
+The following {sandboxed-containers-first} component logs are collected as long as there is at least one pod running with the `kata` runtime:
```

Comment:
maybe it would be more accurate to say something like "...logs are collected for each pod running with the `kata` runtime"